src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedString = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '')
+    .reduce((prev, element) => {
+      const splittedStyle = element.split(':').map((el) => el.trim());
```

Comment:
The `split(':')` method assumes that each style declaration is correctly formatted with a single colon. Consider adding error handling or validation to ensure that the input string is correctly formatted, as malformed input could lead to unexpected behavior.